kernel/mutex.c

```diff
@@ -75,12 +75,8 @@ void _impl_k_mutex_init(struct k_mutex *mutex)
 
 	sys_trace_void(SYS_TRACE_ID_MUTEX_INIT);
 
-	/* initialized upon first use */
-	/* mutex->owner_orig_prio = 0; */
```

Comment:
Surely these comments don't violate MISRA?

Reply:
Actually they do. MISRA 4.4.
Also this code is quite outdated. was used before unified kernel